flaml/nlp/utils.py

```diff
@@ -85,13 +89,19 @@ def tokenize_and_align_labels(
     Y_sent_key=None,
     return_column_name=False,
 ):
+    if hf_args.max_seq_length is not None:
+        logger.warning(
+            "For token classification task, FLAML currently does not support customizing the max_seq_length, so setting max_seq_length={} will remain ineffective.".format(
+                hf_args.max_seq_length
+            )
+        )
     tokenized_inputs = tokenizer(
         [list(examples[X_sent_key])],
         padding="max_length"
         if hf_args and hf_args.pad_to_max_length
         else False,  # to be consistent with https://github.com/huggingface/transformers/blob/main/examples/pytorch/token-classification/run_ner.py#L394
         truncation=True,
-        max_length=hf_args.max_seq_length if hf_args else None,
+        max_length=None,  # hf_args.max_seq_length if hf_args else None,
```

Comment:
This is always set to None, not only for token classification task. This contradicts with the documentation in training_args.py.

Reply:
I think it's within the tokenize_and_align_labels function so only for token classification task.